docs/UserGuide.md

```diff
@@ -559,19 +559,23 @@ Examples:
 
 #### Behaviours of the Fees Calculator
 
-TAB will automatically update your lesson's outstanding fees once the lesson has ended using Fees Calculator feature. 
+TAB will update your lesson's outstanding fees upon launching TAB after the lesson has ended using Fees Calculator feature. Fees will not be updated while TAB is open, it will only update fees upon launch.
```

Comment:
```suggestion
TAB will update your lesson's outstanding fees upon launching TAB after the lesson has ended using the Fees Calculator feature. Fees will not be updated while TAB is open, it will only update fees upon launch.
```